client/my-sites/pages/main.jsx

```diff
@@ -99,7 +99,11 @@ class PagesMain extends React.Component {
 					brandFont
 					className="pages__page-heading"
 					headerText={ translate( 'Pages' ) }
-					subHeaderText={ translate( 'Create, edit, and manage the pages on your site.' ) }
+					subHeaderText={
+						siteId
+							? translate( 'Create, edit, and manage the pages on your site.' )
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.